### ISSUE TITLE
Is it possible to set the random seed for tf.feature_column.categorical_column_with_hash_bucket on operation-level？

### ISSUE THREAD

**Issue by xielm12** [tensorflow/tensorflow#26574]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 1.13.rc1
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
 API:tf.feature_column.categorical_column_with_hash_bucket can not set random seed
**Will this change the current api? How?** 
Yes, It will add a new parametre to set the random seed.
**Who will benefit with this feature?**
The one who want to get two different hash bucket on the same field.
**Any Other info.**

**Comment by ispirmustafa** [comment]
I think it's a reasonable request. Rohan, what do you think?

**Comment by tilakrayal** [comment]
@xielm12,

The **tf.feature_column.categorical_column_with_hash_bucket** api was deprecated. Please take a look at this official doc [link](https://www.tensorflow.org/api_docs/python/tf/feature_column/categorical_column_with_hash_bucket) and image for the reference.
![Screenshot 2023-02-25 8 06 15 PM](https://user-images.githubusercontent.com/81610181/221362726-40f039a8-af6d-4fc6-ace2-c2082feb6d3a.png)



tf.feature_column is not recommended for new code. Instead, feature preprocessing can be done directly using either [Keras preprocessing layers](https://www.tensorflow.org/guide/migrate/migrating_feature_columns) or through the one-stop utility [tf.keras.utils.FeatureSpace](https://www.tensorflow.org/api_docs/python/tf/keras/utils/FeatureSpace) built on top of them. Kindly check the [migration guide](https://tensorflow.org/guide/migrate) for details. Thank you!
